config/application.rb

```diff
@@ -35,5 +35,11 @@ class Application < Rails::Application
         routing_specs: false,
         controller_specs: false
     end
+
+    config.middleware.use config.session_store, config.session_options
+    config.middleware.use Rack::MethodOverride
+    config.middleware.use ActionDispatch::Cookies
+    config.middleware.use ActionDispatch::Session::CookieStore
+    config.middleware.use ActionDispatch::Flash
```

Comment:
`Rack::MethodOverride` was not necessary.
`ActionDispatch::Flash` is a misconfiguration. I set it to not use Flash with devise.
FYI: https://github.com/lynndylanhurley/devise_token_auth/blob/master/README.md
>If using rails-api, you may want to tell devise to not use ActionDispatch::Flash
middleware b/c rails-api does not include it.
See: http://stackoverflow.com/q/19600905/806956
config.navigational_formats = [:json]